banzai_nres/qc/qc_wavelength.py

```diff
@@ -2,48 +2,102 @@
 
 from banzai.stages import Stage
 from banzai.utils import qc
+from astropy import constants as const
+from astropy import units
 from xwavecal.utils.wavelength_utils import find_nearest
+from banzai.utils.stats import robust_standard_deviation
+import logging
 
 from scipy.stats import binned_statistic
 
 
+logger = logging.getLogger('banzai')
+
+
 class AssessWavelengthSolution(Stage):
     """
-    Calculate the dispersion in the wavelength solution and assess whether it is photon-limited or not
+    Calculate the dispersion in the wavelength solution
     @author:mjohnson
```

Comment:
Mirek, you did a lot of work to tidy up the code and make it work better, you should add your name here too!